packages/lib-panoptes-js/src/panoptes.js

```diff
@@ -15,17 +15,17 @@ function checkForAdminFlag () {
   return undefined
 }
 
-// TODO: Write auth client
 // TODO: Consider how to integrate a GraphQL option
-function get (endpoint, query, host) {
+function get (endpoint, query, authorization = '', host) {
   const defaultParams = { admin: checkForAdminFlag(), http_cache: true }
 
   if (!endpoint) return handleMissingParameter('Request needs a defined resource endpoint')
   const apiHost = host || config.host
   const request = superagent.get(`${apiHost}${endpoint}`)
     .set('Content-Type', 'application/json')
     .set('Accept', 'application/vnd.api+json; version=1')
-  // .set('Authorization', apiClient.headers.Authorization);
+
+  if (authorization) request.set('Authorization', authorization)
```

Comment:
I think `Bearer` needs to be prepended to the token string:

```
if (authorization) request.set('Authorization', `Bearer ${authorization}`)
```

Reply:
Ah yep. I forgot about this.